read-fonts/src/tables/glyf.rs

```diff
@@ -812,6 +812,49 @@ mod tests {
     use crate::test_data;
     use crate::{FontRef, GlyphId, TableProvider};
 
+    #[test]
+    fn all_off_curve_to_path() {
+        fn pt(x: i32, y: i32) -> Point<F26Dot6> {
+            Point::new(x, y).map(F26Dot6::from_bits)
+        }
+        let points = [
+            pt(437, 191),
+            pt(716, 191),
+            pt(911, 389),
+            pt(911, 668),
+            pt(716, 864),
+            pt(437, 864),
+            pt(241, 668),
+            pt(241, 389),
+        ];
+        let flags = [PointFlags::off_curve_quad(); 8];
+        let contours = [7];
+        struct SvgPen(String);
+        impl Pen for SvgPen {
+            fn move_to(&mut self, x: f32, y: f32) {
+                self.0.push_str(&format!("M{x:.3},{y:.3} "));
+            }
+            fn line_to(&mut self, x: f32, y: f32) {
+                self.0.push_str(&format!("L{x:.3},{y:.3} "));
+            }
+            fn quad_to(&mut self, cx0: f32, cy0: f32, x: f32, y: f32) {
+                self.0
+                    .push_str(&format!("Q{cx0:.3},{cy0:.3} {x:.3},{y:.3} "));
+            }
+            fn curve_to(&mut self, cx0: f32, cy0: f32, cx1: f32, cy1: f32, x: f32, y: f32) {
+                self.0.push_str(&format!(
+                    "C{cx0:.3},{cy0:.3} {cx1:.3},{cy1:.3} {x:.3},{y:.3} "
+                ));
+            }
+            fn close(&mut self) {
+                self.0.push_str("z ");
+            }
+        }
+        let mut pen = SvgPen(String::default());
+        to_path(&points, &flags, &contours, &mut pen).unwrap();
```

Comment:
I think you need to either choose more distinctive points or comment the test; it's quite hard to tell what  it's meant to detect/prevent if you don't know to begin with.

Fewer points and int points would make it all more readable.

Reply:
I tried to be cheeky by using a contour from the actual outline that caused the failure, but the test was a bit dense.

Reduced the number of points to 4 and made them "rounder" numbers.